### PR TITLE
Update Install.md to add Mac section

### PR DESCRIPTION
**Pull request by conanoc**

### CODE REVIEW

File: doc/INSTALL.md [hunk 1]
```diff
@@ -96,7 +96,43 @@ $ nix-shell -Q -p gdb sqlite autoconf git clang libtool gmp sqlite autoconf \
...
+
+Running lightning:
+
+**Note**: Edit your `~/Library/Application\ Support/Bitcoin/bitcoin.conf`
+to include `rpcuser=<foo>` and `rpcpassword=<bar>` first, you may also
+need to include `testnet=1`
+
+    bitcoind &
+    ./lightningd/lightningd &
+    ./cli/lightning-cli help
```
Comment (cdecker) [inline doc/INSTALL.md]: nit: maybe add a newline here? Some markdown parsers may not get the different indentations.

**Comment by cdecker** [comment]
ACK e3d5b2a

File: doc/INSTALL.md (hunk 1 above)
Comment (conanoc) [reply doc/INSTALL.md]: done.

**Comment by cdecker** [comment]
ACK 276da8d

**Pull request merged**